Exceptions now includes support for Azure Backup

JackStromberg opened the pull request:
https://azure.microsoft.com/en-us/blog/azure-backup-now-supports-storage-accounts-secured-with-azure-storage-firewalls-and-virtual-networks/

azurecla commented:
Hi __@SeanFeldman__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

PRmerger commented:
@SeanFeldman : Thanks for your contribution to the Azure documentation! The author, @tamram, has been notified to review your proposed change.

tamram commented:
Looks good.

#sign-off

KatieCumming commented:
@tamram - Hi Tamra, are you able to add any text under this added release notes heading for 4.5? Thanks, Katie
#hold-off

Saisang commented:
@tysonn please merge this PR.

tamram commented:
@tysonn, can we please merge this? I will update afterwards to add the new text, which I have ready. I don't seem to be able to edit off of this patch.

tamram commented:
@tysonn, Please see PR #30240 (now merged). I have duplicated the change made here (which was to add the header with no content), and also added the necessary text under the header.

@SeanFeldman, thanks for your contribution. You caught our inadvertent omission.

SeanFeldman commented:
@tamram all good :)

KatieCumming commented:
@tamram - closing as discussed as a duplicate. 
@SeanFeldman - the change is live at https://azure.microsoft.com/en-us/documentation/articles/storage-use-emulator/.

SeanFeldman commented:
Great! Thanks @KatieCumming

JackStromberg committed:
Exceptions now includes support for Azure Backup

https://azure.microsoft.com/en-us/blog/azure-backup-now-supports-storage-accounts-secured-with-azure-storage-firewalls-and-virtual-networks/

christopherabrooks commented on articles/storage/common/storage-network-security.md:
```diff
@@ -288,6 +288,7 @@ When the "Trusted Microsoft Services" exception is enabled, the following servic
 
 |Service|Resource Provider Name|Purpose|
 |:------|:---------------------|:------|
+|Azure Backup|Microsoft.Networking|Perform and continue with scheduled and ad-hoc IaaS VM backups and restores.  [Learn more](https://docs.microsoft.com/en-us/azure/backup/backup-introduction-to-azure-backup).|
```
looks like the second column here is wrong - can you add the Resource Provider name for Microsoft Backup here?

christopherabrooks replied:
@AlanSt , FYI

JackStromberg committed:
Fixed Resource Provider and removed notice of limitation

Updated resource provider to Microsoft.Backup for Azure Backup exception.
Changed note to remove verbiage on limitation of Azure Backup with firewall rules

JackStromberg reviewed (commented):
Made changes per comments in other thread

christopherabrooks commented on articles/storage/common/storage-network-security.md:
```diff
@@ -288,6 +284,7 @@ When the "Trusted Microsoft Services" exception is enabled, the following servic
 
 |Service|Resource Provider Name|Purpose|
 |:------|:---------------------|:------|
+|Azure Backup|Microsoft.Backup|Perform and continue with scheduled and ad-hoc IaaS VM backups and restores.  [Learn more](https://docs.microsoft.com/en-us/azure/backup/backup-introduction-to-azure-backup).|
```
Is the exception required only for VM's using UNMANAGED disks?  If so, can we call that out in the purpose here?

JackStromberg replied:
We could call it out, but wouldn't that statement be non-applicable to the article since the article is about storage account firewall rules?  Just want to clarify it's not a limitation since it's not applicable to create firewall rules for managed disks as they aren't placed in customer owned storage accounts.

christopherabrooks replied:
How about "Perform backups and restores of un-managed disks in IAAS virtual machines. (not required for managed disks).  [Learn More]"

JackStromberg committed:
Updated verbiage clarifing Azure Backup supports unmanaged storage

Updated verbiage clarifing Azure Backup supports unmanaged storage; managed disks are not applicable to this article.

JackStromberg reviewed (commented):
Updated verbiage

JackStromberg committed:
Updated tabel verbiage
